lib/types/api.types.ts

```diff
@@ -271,56 +292,70 @@ export type DialogAppSDK = Omit<BaseAppSDK, 'ids'> & {
   window: WindowAPI
 }
 
-export type PageAppSDK = Omit<BaseAppSDK, 'ids'> & {
+export type PageAppSDK<
+  InstallationParameters extends KeyValueMap = KeyValueMap,
+  InstanceParameters extends KeyValueMap = KeyValueMap,
+  InvocationParameters extends SerializedJSONValue = { path: string }
+> = Omit<BaseAppSDK<InstallationParameters, InstanceParameters, InvocationParameters>, 'ids'> & {
```

Comment:
Page location has no instance params and the invocation params are fixed

```suggestion
export type PageAppSDK<InstallationParameters extends KeyValueMap = KeyValueMap> = Omit<
  BaseAppSDK<InstallationParameters, never, { path: string }>,
  'ids'
> & {
```